We use backports for driver updates - make sure we can compile in the glue code regardless

Signed-off-by: Imre Kaloz <kaloz@openwrt.org>

--- a/drivers/net/wireless/ti/Kconfig
+++ b/drivers/net/wireless/ti/Kconfig
@@ -15,7 +15,7 @@ source "drivers/net/wireless/ti/wlcore/K
 
 config WILINK_PLATFORM_DATA
 	bool "TI WiLink platform data"
-	depends on WLCORE_SDIO || WL1251_SDIO
+	depends on WLCORE_SDIO || WL1251_SDIO || ARCH_OMAP2PLUS
 	default y
 	---help---
 	Small platform data bit needed to pass data to the sdio modules.
